torch/onnx/symbolic.py

```diff
@@ -230,7 +230,7 @@ def _try_get_scalar_type(*args):
 
 # used to represent "missing" optional inputs
 def unused(g):
-    return g.op("prim::None")
+    return g.op("prim::Constant")
```

Comment:
Type is not set correctly here!

Reply:
Do you know how to do that? I've looked around but without success

Reply:
setType should be bound on the returned Value object. It might be necessary to add OptionalType to the python bindings to construct one.